# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../sysfsutils/sysfsutils-0001-libsysfs.pc.patch
# Copyright (C) 2013 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./configure.ac.orig	2013-01-18 10:38:17.773116875 +0100
+++ ./configure.ac	2013-01-18 10:39:11.996497255 +0100
@@ -47,6 +47,7 @@
 AC_FUNC_STAT
 AC_CHECK_FUNCS([bzero isascii memset strchr strerror strrchr strstr strtol])
 
+AC_CONFIG_FILES([libsysfs.pc])
 AC_CONFIG_FILES([Makefile
                  lib/Makefile
                  cmd/Makefile
--- ./Makefile.am.orig	2005-12-06 10:03:17.000000000 +0100
+++ ./Makefile.am	2013-01-18 10:33:46.096230349 +0100
@@ -4,6 +4,9 @@
 includedir=@includedir@/sysfs
 include_HEADERS = include/libsysfs.h include/dlist.h
 
+pkgconfigdir = $(libdir)/pkgconfig
+pkgconfig_DATA = libsysfs.pc
+
 dist-hook:
 	rm -rf `find $(distdir)/docs -name CVS`
 	rm -rf `find $(distdir)/include -name CVS`
--- /dev/null	2013-01-18 10:33:46.096230349 +0100
+++ ./libsysfs.pc.in	2013-01-18 10:43:27.806457538 +0100
@@ -0,0 +1,10 @@
+prefix=@prefix@
+exec_prefix=@prefix@
+libdir=/lib
+includedir=@prefix@/include/sysfs/
+
+Name: libsysfs
+Description: interface to sysfs
+Version: @PACKAGE_VERSION@
+Libs: -L${libdir} -lsysfs
+Cflags: -I${includedir}
